client/src/shared/styles/emotion.d.ts

```diff
@@ -0,0 +1,8 @@
+import { ColorType } from './colors';
+
+declare module '@emotion/react' {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+  export interface Theme {
+    colors: ColorType;
+  }
```

Comment:
p1. 
interface 대신 type 사용!!
```suggestion
  export type Theme = {
    colors: colorType
  }
```

Reply:
오 그런 차이가 있군요?? 배워갑니다